README.md

```diff
@@ -129,27 +129,18 @@ The easiest way to run all components of Saleor (API, storefront, and dashboard)
 
 ## Storefront
 
-An open-source storefront in React.js with Next.js. Built for Headless Commerce, using a modern stack with TypeScript, GraphQL, Apollo, and Tailwind CSS.
+An Open Source storefront example built with Next.js App Router, React.js, TypeScript, GraphQL, and Tailwind CSS.
```

Comment:
Open Source or open-source? We use two different ways of writing that in this file.

Reply:
Open Source is written separately and capitalized when it refers to the Open Source Initiative as a noun. It's hyphenated in lowercase when used as an adjective (which I believe is the case here).

Reply:
Okay, thank you! I'll update it here (and elsewhere if I find any occurrences)